Description: Disable confusing (to users) warning about deprecated schema paths
Author: Iain Lane <iain.lane@canonical.com>
Forwarded: not-needed

Index: b/gio/glib-compile-schemas.c
===================================================================
--- a/gio/glib-compile-schemas.c
+++ b/gio/glib-compile-schemas.c
@@ -1219,6 +1219,9 @@
       return;
     }
 
+  // Disable this warning: it confuses users and there is unlikely to be much
+  // progress towards fixing
+  /*
   if (path && (g_str_has_prefix (path, "/apps/") ||
                g_str_has_prefix (path, "/desktop/") ||
                g_str_has_prefix (path, "/system/")))
@@ -1231,6 +1234,7 @@
       g_printerr ("%s\n", message);
       g_free (message);
     }
+    */
 
   state->schema_state = schema_state_new (path, gettext_domain,
                                           extends, extends_name, list_of);
